imagedephi/templates/BaseSelectorWidget.html.j2

```diff
@@ -34,8 +34,14 @@
 <ul>
   {% for child_image in directory_data.child_images[:10] %}
     <li>
-      <i class="mdi mdi-image text-sky-800"></i>
-      {{ child_image.name }}
+      <div class="inline-flex">
+        <img src="{{ image_url(child_image.name, 'label') }}" alt="label image" />
+        <img src="{{ image_url(child_image.name, 'macro') }}" alt="macro image" />
+        <img src="{{ image_url(child_image.name, 'thumbnail') }}"
+             alt="thumbnail image" />
+        <i class="mdi mdi-image text-sky-800"></i>
+        {{ child_image.name }}
+      </div>
```

Comment:
I expect to remove this before the PR is merged, but this could be useful for reviewers testing the current GUI using the new image endpoints.

Reply:
This was handy.  It shows that on TCGA-5N-A9KM-01Z-00-DX1.5197F750-D17F-459B-B74D-846F5F50F7B7.svs I'd expect to get a label image, but don't.  Overall, this PR looks good.

Reply:
You can find a copy of the mentioned file on my local server under /collection/TCGA/square

Reply:
I'm a little bit confused about this image. There is indeed a label image, (Directory 5 when looking at the output of `tifftools dump`) but it doesn't have an `ImageDescription` tag pointing that out. 

The value of `NewSubfileType` for this directory is `1`, but that seems odd. According to tifftools, this bit represents "Image is a reduced-resolution version of another image in this TIFF file," but the label image isn't a reduced resolution version of any image in the file (that I'm aware of).

Additionally, other directories in the image, apart from the label and macro images, have `NewSubfileType = 0`. I would expect all but the highest resolution image to hava a value of `1`. What's going on here? Is there another heuristic for finding a label image among the IFDs? If large image does this well, could you point me in that direction?

Reply:
Based on our conversation on Tuesday, I updated the logic in ad0302d